### PR TITLE
 [Translation] TransChoice invalide plural translation

### PR DESCRIPTION
**Pull request by Simperfit**
This is linked to the new feature at https://github.com/symfony/symfony/pull/31110

### CODE REVIEW

**Comment by Simperfit** [comment]
Needs update after code review ping @Simperfit

**Comment by Simperfit** [comment]
Status: Needs Work

File: translation.rst [hunk 1]
```diff
@@ -230,11 +230,18 @@ method or the ``transchoice`` tag/filter in your :ref:`template <translation-tag
 For much more information, see :ref:`component-translation-pluralization`
 in the Translation component documentation.
 
+
```
Comment (OskarStark) [inline translation.rst]: This blank line should be removed

File: translation.rst [hunk 2]
```diff
@@ -230,11 +230,18 @@ method or the ``transchoice`` tag/filter in your :ref:`template <translation-tag
...
 .. deprecated:: 4.2
 
     In Symfony 4.2 the ``Translator::transChoice()`` method was deprecated in
     favor of using ``Translator::trans()`` with ``%count%`` as the parameter
     driving plurals.
+    
+
```
Comment (OskarStark) [inline translation.rst]: This blank line should be removed

**Comment by maxhelias** [comment]
This PR is outdated 😉

**Comment by HeahDude** [comment]
Closing here as the related PR has been closed. Thanks!

**Pull request closed without merging**